base/system/winlogon/resource.h

```diff
@@ -32,3 +38,6 @@
 #define IDS_REACTOSISSTARTINGUP          1690
 
 #define IDS_FAILEDACTIVATEUSERSHELL 1001
+
+/* Timers */
+#define IDT_SYSSHUTDOWN 2000
```

Comment:
A window timer is not really a resource, so for this one, I would tend to prefer to define it inside the .c file where you use it.